src/views/catalog/CreateFromInstanceTypes/components/VMDetailsSection/components/VMDetailsBody/VMDetailsBody.scss

```diff
@@ -0,0 +1,3 @@
+.instancetypes-vm-details-body {
+  margin-top: var(--pf-global--spacer--lg);
+}
```

Comment:
Same as in the other css file ;) 
+1 for using Patternfly variables!

Reply:
Done!